src/components/TabRouter.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { TouchableOpacity, View } from "react-native";
+
+const TabRouter = (props: {
+  activeTabIndex: number;
+  setActiveTab: any;
+  bottomTab: boolean;
+  tabButtons: any[]
+  tabViews: any[];
+  viewHeightReduction: number;
+}) => {  
+  if(props.tabViews.length === 1){
+    return props.tabViews[0];
+  }
+
+  const tabBar = (
+    <View
+      style={{
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+        height: 52
+      }}
+    >
+      {props.tabButtons.map((button, index) => (
+        <TouchableOpacity onPress={() => props.setActiveTab(index)}>
+          {button}
+        </TouchableOpacity>
+      ))}
+    </View>
+  );
+
+  const tabView = (
+    <View style={{ height: `Calc(100vh - ${props.viewHeightReduction}px` }}>
+      {props.tabViews[props.activeTabIndex]}
+    </View>
+  );
```

Comment:
Can we give tabs a selected state so I know which tab I am on?

Reply:
^^. Maybe even an active and inactive color prop for each? That will definitely be a thing Colton wants.